contents/customers/significa.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Why Significa switched from Plausible to PostHog
+customer: Significa
+logo: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/customers/4dayweek/4dayweek-logo.png
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/customers/4dayweek/4dayweek-logo-dark.png
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1710950177/posthog.com/contents/images/customers/significa/significa.png
+industries:
+  - Agency
+users:
+  - Marketing
+toolsUsed:
+  - Web analytics
+  - Product analytics
+date: 2024-03-20
+---
+
+[Significa](https://significa.co/) is an award-winning digital-led agency based in the Portuguese city of Porto. Since 2015, Significa has worked with clients such as Dia (for which [it won a Red Dot Award](https://significa.co/projects/dia)), Zoko (for which [it won a Good Design Award](https://significa.co/projects/zoko)), and Passaporte Natura 2000 (for which [it won an iF Design award](https://significa.co/projects/passaporte-natura))
```

Comment:
```suggestion
[Significa](https://significa.co/) is an award-winning digital-led agency based in the Portuguese city of Porto. Since 2015, Significa has worked with such as TFP, for whom it built Dia ([a Red Dot Award winner](https://significa.co/projects/dia)), Zoko ([winner of a Good Design Award](https://significa.co/projects/zoko)), and Passaporte Natura 2000 ([iF Design award winner](https://significa.co/projects/passaporte-natura)).
```